intl/messages/cs.json

```
@@ -89,7 +89,7 @@
 
   "gettingStarted": {
     "sectionTitle": "Začínáme",
-    "sectionDesc": "IPFS vám dává primitivní řešení pro všechna vaše data na DWebu. Data můžete zpřístupnit do sítě nebo načíst stávající data prostřednictvím svého CID, identifikátoru obsahu. Vyzkoušejte to níže nebo si prohlédněte naše [ProtoSchool návody] (https://proto.school/#/tutorials?course=ipfs)!",
+    "sectionDesc": "IPFS vám dává primitivní řešení pro všechna vaše data na DWebu. Data můžete zpřístupnit do sítě nebo načíst stávající data prostřednictvím svého CID, identifikátoru obsahu. Vyzkoušejte to níže nebo si prohlédněte naše [ProtoSchool návody] (https://proto.school/course/ipfs)!",
```

Not sure if this effects the rendering or not but there shouldn't be whitespace between ] and (.
```suggestion
    "sectionDesc": "IPFS vám dává primitivní řešení pro všechna vaše data na DWebu. Data můžete zpřístupnit do sítě nebo načíst stávající data prostřednictvím svého CID, identifikátoru obsahu. Vyzkoušejte to níže nebo si prohlédněte naše [ProtoSchool návody](https://proto.school/course/ipfs)!",
```